examples/simple_box/src/client.rs

```diff
@@ -121,16 +125,24 @@ pub(crate) fn buffer_input(
         direction.right = true;
     }
     if !direction.is_none() {
-        return connection_manager.add_input(Inputs::Direction(direction), tick);
+        input = Inputs::Direction(direction);
     }
     if keypress.pressed(KeyCode::Backspace) {
-        // currently, inputs is an enum and we can only add one input per tick
-        return connection_manager.add_input(Inputs::Delete, tick);
+        input = Inputs::Delete;
     }
     if keypress.pressed(KeyCode::Space) {
-        return connection_manager.add_input(Inputs::Spawn, tick);
+        input = Inputs::Spawn;
+    }
+    // if we run in listen-server mode and the server/client are running in the same app
+    // emit the input event directly to the server
+    if let Some(mut server_input_event) = server_input_event {
+        info!(?tick, "send client input");
+        // TODO: should we still add the event to the buffer, for rollbacks?
+        //  maybe not, because there cannot be mispredictions?
+        let _ = server_input_event.send(server::InputEvent::new(Some(input), client_id));
```

Comment:
As above, I think it's probably the nicest if there's one abstraction that captures both cases